datalake/src/main/java/com/sequenceiq/datalake/service/sdx/StackRequestManifester.java

```diff
@@ -129,6 +136,28 @@ private void setupAuthentication(DetailedEnvironmentResponse environment, StackV
         }
     }
 
+    private void setupSecurityAccess(DetailedEnvironmentResponse environment, StackV4Request stackRequest) {
+        List<InstanceGroupV4Request> instanceGroups = stackRequest.getInstanceGroups();
+        if (instanceGroups != null) {
+            instanceGroups.stream()
+                    .filter(ig -> ig.getType() == InstanceGroupType.GATEWAY)
```

Comment:
I think we only prepare here the gateway group. @bergerdenes

Reply:
changed as requested to update Core instance group as well